src/pudl/metadata/resources/ferc1_eia_record_linkage.py

```diff
@@ -4,15 +4,16 @@
 
 RESOURCE_METADATA: dict[str, dict[str, Any]] = {
     "out_eia__yearly_plant_parts": {
-        "description": """Output table with the aggregation of all EIA plant parts. For use with matching to FERC 1.
-
-Practically speaking, a plant is a collection of generator(s). There are many
-attributes of generators (i.e. prime mover, primary fuel source, technology
-type). We can use these generator attributes to group generator records into
-larger aggregate records which we call "plant parts". A plant part is a record
-which corresponds to a particular collection of generators that all share an
-identical attribute and utility owner. E.g. all of the generators with unit_id=2, or all
-of the generators with coal as their primary fuel source.
+        "description": {
+            "additional_summary_text": "the aggregation of all EIA plant parts, for use in matching to FERC 1.",
+            "additional_details_text": (
```

Comment:
its organized duplication but omigosh so much duplication
```suggestion
            "additional_summary_text": "the aggregation of all EIA plant parts, for use in matching to FERC 1.",
            "usage_warnings": ["aggregation_hazard"],
            "additional_details_text": (
```

Reply:
also i agree with your note in sheet that "the aggregation" doesn't really make sense.  I think removing "the aggregation of" makes sense to me even though as a summary line not knowing what the heck a plant part is is... a little weird but fine